app/models/absolute_identifier.rb

```diff
@@ -15,22 +16,22 @@
 #  created_at          :datetime         not null
 #  updated_at          :datetime         not null
 #  batch_id            :bigint
+#  holding_id          :string
 #
 # Indexes
 #
 #  absolute_identifiers_uniqueness         (prefix,suffix,pool_identifier) UNIQUE
 #  index_absolute_identifiers_on_batch_id  (batch_id)
 #
-# Foreign Keys
-#
-#  fk_rails_...  (batch_id => batches.id)
-#
 class AbsoluteIdentifier < ApplicationRecord
```

Comment:
We should look into adding a validation so that we can only build AbIDs for records which are in designated locations.

Reply:
AbIDs now have to be in the "rare" location to be associated with a barcode.